src/test/scala/chiselTests/MixedVecSpec.scala

```diff
@@ -293,7 +293,7 @@ class MixedVecSpec extends ChiselPropSpec with Utils {
       io.outMono := (io.inMono: Data)
       io.outBi <> io.inBi
     })
-    chirrtl should include("io.outMono <= io.inMono @[src/test/scala/chiselTests/MixedVecSpec.scala")
-    chirrtl should include("io.outBi <= io.inBi @[src/test/scala/chiselTests/MixedVecSpec.scala")
+    chirrtl should include("io.outMono <= io.inMono @")
+    chirrtl should include("io.outBi <= io.inBi @")
```

Comment:
Consider further shortening to `io.inMono"` (ignoring a check of the source locator as that doesn't seem to be relevant to the test).